### PR TITLE
Add Magnet.me

### PR DESCRIPTION
**Pull request by rogierslag**
Screenshot 2023-03-21 at 00 52 11

Refs https://github.com/2factorauth/twofactorauth/pull/6706

### CODE REVIEW

File: entries/m/magnet.me.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Magnet.me": {
+    "domain": "magnet.me",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
```
Comment (hkamran80) [inline entries/m/magnet.me.json]:
```suggestion
    "categories": [
```

File: img/m/magnet.me.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg height="37" viewBox="0 0 37 37" width="37" xmlns="http://www.w3.org/2000/svg"><path d="m4 4h29v29h-29z" fill="#fff"/><path d="m34 0h-31c-3 0-3 3-3 3v31c0 2.9 3 3 3 3h31c2.9 0 3-3 3-3v-31c0-3-3-3-3-3zm-23 29h-5v-4h5zm10 0h-5v-4h5zm10 0h-5v-4h5zm0-7h-5v-6s0-2.5-2.4-2.5c-2.6 0-2.6 2.5-2.6 2.5v6h-5v-6s0-2.5-2.5-2.5-2.5 2.5-2.5 2.5v6h-5v-13s.1-1 1-1h6.5s3.8 0 5 2.5c1.2-2.5 5-2.5 5-2.5 7.5 0 7.5 8 7.5 8z" fill="#ff5c51" fill-rule="evenodd"/></svg>
```
Comment (hkamran80) [inline img/m/magnet.me.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 37 37"><path fill="#fff" d="M4 4h29v29H4z"/><path fill="#ff5c51" d="M34 0H3C0 0 0 3 0 3v31c0 2.9 3 3 3 3h31c2.9 0 3-3 3-3V3c0-3-3-3-3-3zM11 29H6v-4h5zm10 0h-5v-4h5zm10 0h-5v-4h5zm0-7h-5v-6s0-2.5-2.4-2.5C21 13.5 21 16 21 16v6h-5v-6s0-2.5-2.5-2.5S11 16 11 16v6H6V9s.1-1 1-1h6.5s3.8 0 5 2.5c1.2-2.5 5-2.5 5-2.5C31 8 31 16 31 16z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Can you provide evidence of 2FA?

**Comment by rogierslag** [comment]
Hopefully this GIF of logging in with 2FA support suffices. If not, please let me know and I'd be happy to provide what's needed

**Comment by hkamran80** [comment]
Could you sign in to CircleCI again? Sometimes when a PR author has used CircleCI previously, CircleCI tries to use the author's credentials instead of 2factorauth's. If the PR author's authentication token has expired the build can't run and so the auth token needs to be regenerated by signing in to CircleCI.

**Comment by rogierslag** [comment]
Done! Amended the commit twice to force a re-run of CircleCI

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**